app/models/planning_application.rb

```diff
@@ -659,6 +659,10 @@ def longitude
     super || lonlat.try(:x)
   end
 
+  def lat_lon_zoom(zoom = "17")
```

Comment:
This is a little pedantic of me but maybe this is better off in the helper, since it's only used by the helper. That way we can keep view code out of the model itself.

Reply:
Yeah on reflection is quite specific to that url generation so not sure we'd reuse elsewhere